app/uk/gov/hmrc/eeittadminfrontend/services/AuthService.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.eeittadminfrontend.services
+
+import cats.data.Validated
+import play.api.Configuration
+import uk.gov.hmrc.eeittadminfrontend.models.{Email, LoginError}
+
+class AuthService(config: Configuration) {
+
+  def checkUser(email: Email): Validated[LoginError, Unit] = {
```

Comment:
Have fun :) We can discuss later what are trade-offs using validated/either/or custom validated which happens for example [in gform](https://github.com/hmrc/gform/blob/349a3a1c0156ce24845f173a054e780d22baaed7/app/uk/gov/hmrc/gform/core/ValidationResult.scala).

As a quick hint - Validated are good for collecting error messages but you need to use applicative builder in order to have it.